components/merkledb/src/views/metadata.rs

```diff
@@ -344,6 +345,19 @@ impl<T: RawAccess> IndexesPool<T> {
         self.set_len(len + 1);
         (metadata, is_phantom)
     }
+
+    pub(crate) fn suffixes<K: BinaryKey + ?Sized>(&self, prefix: &str) -> Vec<K::Owned> {
+        let prefix = concat_keys!(prefix, INDEX_NAME_SEPARATOR);
```

Comment:
Nit: I'd rather have the prefix logic encapsulated in the `views::address` module, as in #1637.